assemblies/assembly_openshift-authoring-managed/main.adoc

```diff
@@ -26,7 +26,7 @@ and other business assets.
 * The OpenShift project for the deployment has been created.
 * You are logged in to the project using the `oc` command. For more information about the `oc` command-line tool, see the OpenShift https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html-single/cli_reference/[CLI Reference]. If you want to use the OpenShift Web console to deploy templates, you must also be logged on using the Web console.
 * Dynamic persistent volume (PV) provisioning is enabled. Alternatively, if dynamic PV provisioning is not enabled, a sufficient persistent volume must be available. By default, {CENTRAL} requires one 1Gi PV. You can change the PV size for {CENTRAL} persistent storage in the template parameters.
-* Your OpenShift environment supports persistent volumes with `ReadWriteMany` mode. For information about access mode support in OpenShift Online volume plug-ins, see https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html-single/architecture/#pv-access-modes[Access Modes].
+* If you intend to deploy a high-availability authoring environment, which includes high-availability {CENTRAL}, your OpenShift environment supports persistent volumes with `ReadWriteMany` mode. If your environment does not support this mode, you can use NFS to provision the volumes. However, for best performance and reliability use GlusterFS to provision persistent volumes for a high-availability authoring environment. For information about access mode support in OpenShift Online volume plug-ins, see https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html-single/architecture/#pv-access-modes[Access Modes].
```

Comment:
> your OpenShift environment supports

your OpenShift environment **needs to** support?

Otherwise looks good to me, but not sure what you're trying to say here. That OCP can support RWX? I suppose that depends on the details of the environment.